tests/unit/test_addonHandler/test_addonBundle.py

```diff
@@ -0,0 +1,575 @@
+# A part of NonVisual Desktop Access (NVDA)
```

Comment:
Note: this file has not been fully reviewed - however the rest of this PR has been